test/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "bedrock-payment-paypal-test-project",
+  "version": "0.0.1",
+  "description": "Integration Tests for Bedrock Payment PayPal.",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "test": "node --preserve-symlinks test test"
+  },
+  "keywords": [
+    "test",
+    "bedrock-payment",
+    "bedrock-payment-paypal"
+  ],
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com"
+  },
+  "license": "BSD-3-Clause",
```

Comment:
Not accurate, most test package files do not include license at all.

Reply:
Thanks for checking this. Addressed here: https://github.com/digitalbazaar/bedrock-payment-paypal/pull/5/commits/62e503feebde65c71ead6d0996b91d4abc28df23